src/Microsoft.CSharp/src/Configurations.props

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netstandard1.7|AnyOS;
```

Comment:
If we change them then we'll need to add the UWP configs as well.  It seems reasonable to do this for anything that is part of NETStandard, if the project currently targets netstandard, change it to target the full set of frameworks we want to build in CoreFx.